fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biome/MixinAddHillsLayer.java

```diff
@@ -48,9 +55,13 @@ private void sample(LayerRandomnessSource rand, LayerSampler biomeSampler, Layer
 		final int biomeId = biomeSampler.sample(chunkX, chunkZ);
 		int noiseSample = noiseSampler.sample(chunkX, chunkZ);
 		int processedNoiseSample = (noiseSample - 2) % 29;
-		final Biome biome = BuiltinRegistries.BIOME.get(biomeId);
+		RegistryKey<Biome> key = Biomes.fromRawId(biomeId);
```

Comment:
I would suggest using `InternalBiomeUtils.fromRawId` similar to `InternalBiomeUtils.getRawId` so we can make them both use a `DynamicRegistryManager` if needed.

Reply:
Answered above.